cdap-explore/src/test/java/co/cask/cdap/explore/service/BaseHiveExploreServiceTest.java

```diff
@@ -138,24 +144,23 @@
 
   protected static Injector injector;
 
-  @ClassRule
-  public static TemporaryFolder tmpFolder = new TemporaryFolder();
-
-  protected static void initialize() throws Exception {
-    initialize(CConfiguration.create());
+  protected static void initialize(TemporaryFolder tmpFolder) throws Exception {
```

Comment:
I'm now passing in `TemporaryFolder` in initialize, otherwise when we call `intitalize` from `BaseHiveExploreServiceTest` the `@ClassRule` may not be initiailzed.